mods/ra/rules/ai.yaml

```diff
@@ -1,7 +1,9 @@
 Player:
+	HarvesterBotModule:
 	HackyAI@RushAI:
 		Name: Rush AI
 		Type: rush
+		Modules: harvester-module
```

Comment:
Why do we also have to list the modules on the `HackyAI`?

Is their existence on `Player` not enough to find and use them?

Reply:
Different bots may want to use different instances of the same trait, to provide different effects ingame.

Reply:
Hmm okay.
Do we have a good place to document / explain this usecase?

It is obviously completely valid but I'd prefer we explain it explicitly somewhere game authros can find on their own.

Reply:
Maybe we could write a wiki entry about this once all the main modules are merged into bleed.

That being said,
> Different bots may want to use different instances of the same trait, to provide different effects ingame.

This will become more obvious once we get to the BaseManager and SquadManager modules, because those will actually differ for each AI in the shipping mods and therefore make use of this.

Reply:
Removed Modules/Name fields in favor of using conditions.